programs/openbook-v2/src/state/orderbook/book.rs

```diff
@@ -19,6 +19,13 @@ pub struct Orderbook<'a> {
     pub asks: RefMut<'a, BookSide>,
 }
 
+pub struct TakenQuantitiesIncludingFees {
```

Comment:
i think a more descriptive name should be used here, or at least not include the order_Id